src/ExtensionConfigHandlerInterface.php

```diff
@@ -107,6 +108,47 @@ public function getStorageComparer(array $source_dirs, $subdir = InstallStorage:
   public function importFromComparer(StorageComparerInterface $storage_comparer);
 
   /**
+   * Get the config dependancies for an extension.
+   *
+   * For the config currently exported to a project, find the config
+   * dependencies required for it to work.
+   *
+   * @param string $extension
+   *   The machine name of the project to get the config dependencies for.
+   * @return array
+   *   An array of config the extension depends on.
+   */
+  public function getConfigDependancies($extension);
```

Comment:
Okidoke - I think either spelling is actually valid, but happy to standardise on "depenent" as the adjective and "dependant" as the noun.